src/context/cart-context.js

```diff
@@ -0,0 +1,84 @@
+import axios from "axios";
+import react, { useContext, createContext, useState } from "react";
+
+const CartContext = createContext();
+const Token = localStorage.getItem("Manazone.Token");
+const CartProvider = ({ children }) => {
+  const [cartData, setCartData] = useState([]);
+  const addToCart = async (item) => {
+    try {
+      const response = await axios.post(
+        "/api/user/cart",
```

Comment:
one suggestion here : You can add one file for backend API urls so if you want to reuse it you can directly imnport them from that file. Or in future if any url changes then you can change it  in one file and it will reflect in all places